client/my-sites/themes/auto-loading-homepage-modal.scss

```diff
@@ -1,138 +1,65 @@
 @import "@wordpress/base-styles/mixins";
 
-.themes__auto-loading-homepage-modal {
+.dialog__content.themes__auto-loading-homepage-modal {
 	position: relative;
-
-	h1 {
-		margin-bottom: 1em;
-		text-align: center;
-	}
-}
-
-.themes__auto-loading-homepage-modal-close-icon {
-	cursor: pointer;
-	width: 16px;
-	height: 16px;
-	position: absolute;
-	top: 10px;
-	right: 10px;
-
-	@include break-medium {
-		width: 24px;
-		height: 24px;
-		right: 20px;
-		top: 20px;
+	padding: 45px 45px 17px;
+	max-width: 640px;
+	box-sizing: border-box;
+	.auto-loading-homepage-modal__heading {
+		@extend .wp-brand-font;
+		font-size: $font-headline-small;
+		line-height: 1.1;
+		width: 100%;
+		margin-bottom: 15px;
 	}
 }
 
-.themes__auto-loading-homepage-modal-homepage-hint {
-	margin-left: 1.5em;
-	font-size: $font-body-small;
-	font-style: italic;
-	margin-top: -5px;
-}
-
 .themes__theme-preview-wrapper {
 	display: flex;
 	flex-direction: column;
-	align-items: center;
-	max-width: 700px;
 }
 
-.themes__autoloading-homepage-option-description {
-	font-size: $font-body-small;
-	min-height: 90px;
-
-	@include break-medium {
-		width: 70%;
-		min-height: 0;
-	}
+.themes__auto-loading-homepage-modal-close-icon {
+	cursor: pointer;
+	color: var(--color-text-subtle);
+	position: absolute;
+	top: 0;
+	right: 8px;
 }
 
-.themes__theme-preview-items {
-	display: flex;
-	flex-direction: column;
-	margin: 0 0 1em;
-	max-width: 700px;
-
-	@include break-medium {
-		flex-direction: row;
-	}
+.auto-loading-homepage-modal__description {
+	margin-bottom: 30px;
 }
 
-.themes__theme-preview-item {
-	img,
-	.themes__iframe-wrapper {
-		display: none;
-	}
-
-	@include break-medium {
-		border: 1px solid var(--color-border-subtle);
-		border-radius: 2px;
-		width: 48%;
-		margin-left: 2%;
-
-		img,
-		.themes__iframe-wrapper,
-		.themes__theme-preview-image-wrapper {
-			display: block;
-			height: 100%;
-		}
-
-		.themes__iframe-wrapper,
-		.themes__theme-preview-image-wrapper {
-			max-height: 235px;
-			overflow: hidden;
-		}
-
-		.form-label {
-			height: 100%;
-		}
-
-		.form-radio {
-			margin: 1em;
-		}
-
-		.form-radio__label {
-			margin: 1em 1em 1em 3em;
-		}
-
-		&:first-of-type {
-			margin-left: 0;
-			margin-right: 2%;
-		}
-	}
-
-	img {
-		width: 100%;
-		height: auto;
+.auto-loading-homepage-modal__checkbox {
+	margin-bottom: 52px;
+	.components-checkbox-control__label {
+		font-size: $font-body;
 	}
 }
 
-.themes__theme-preview-item-iframe-container {
+.auto-loading-homepage-modal__actions {
 	display: flex;
-	flex-direction: column;
-
-	.themes__iframe-wrapper {
-		flex-grow: 1;
-		overflow: hidden;
-		position: relative;
-		pointer-events: none;
-	}
-	iframe {
-		height: 357%;
-		width: 357%;
-		max-width: 357%;
-		transform: scale(0.28);
-		transform-origin: top left;
-		position: absolute;
-		top: 0;
-		left: 0;
+	justify-content: flex-end;
+
+	.button {
+		padding: 9px 25px;
+		border-radius: 4px;
+		font-weight: 500;
+		&.is-primary {
```

Comment:
We might need to also handle the disabled (and hover) state, otherwise it looks odd:

![Screenshot 2023-06-09 at 4 21 31 PM](https://github.com/Automattic/wp-calypso/assets/797888/578a40b1-50ca-488c-a610-d8ee601bf90a)

```suggestion
		&.is-primary:not([disabled]) {
```

Reply:
I addressed this in https://github.com/Automattic/wp-calypso/pull/77881/commits/a9b46d064777898b23a028cc6f3fa272b10d55af 🙏 Let me know if I took it wrong.
[![Image from Gyazo](https://i.gyazo.com/21e46f6332e5c1f1550c669b4d676290.gif)](https://gyazo.com/21e46f6332e5c1f1550c669b4d676290)

Reply:
I guess the disabled styles might be as followed:

![image](https://github.com/Automattic/wp-calypso/assets/13596067/61d59823-c689-4dd9-88b9-7a6722eee592)

Reply:
Thanks, @arthur791004! Where can I find that style as an example?

Reply:
Maybe this: Tv3pYqA3EcRfiXC31IxrXE-fi-716_25378

Reply:
Addressed it in this way 👍
<img width="700" alt="Screen Shot 2023-06-12 at 15 27 30" src="https://github.com/Automattic/wp-calypso/assets/5287479/7d591e0b-f1de-4a2f-8c13-26eb84071b46">